tasks/sync-granule/tests/sync_granule_test.js

```diff
@@ -1062,6 +1062,54 @@ test.serial('download multiple granules from S3 provider to staging directory',
   }
 });
 
+test.serial('download() ingests multiple granules from S3 provider to staging directory with ACL', async (t) => {
+  t.context.event.input.granules = t.context.event_multigran.input.granules;
+  try {
+    await prepareS3DownloadEvent(t);
+    t.context.event.config.ACL = 'disabled';
```

Comment:
Setting ACL here. Not sure if this test is necessary and maybe it should be combined with the test here https://github.com/nasa/cumulus/pull/3104/files#diff-80939d13722e8a8ef23dfa696e83b3361b1d72ea586b7c54dd5d38cf52ef1fecR1024 ?

Reply:
Yes, I am suggesting to combine it with `download multiple granules from S3 provider to staging directory` test